docs/en/05_Contributing/02_Build_Tooling.md

```diff
@@ -12,6 +12,9 @@ The [NodeJS](https://nodejs.org) JavaScript runtime is the foundation of our cli
 build tool chain. If you want to do things like upgrade dependencies, make changes to core
 JavaScript or SCSS files, you'll need Node installed on your dev environment. Our build
 tooling supports the v6.x (LTS) version of NodeJS.
+
+SilverStripe authored npm dependencies are posted under the
+[silverstripe npm](https://www.npmjs.com/org/silverstripe) organisation.
```

Comment:
this seems out of place, the mention in the Javascript coding convention seems enough :)